db/seeds.rb

```diff
@@ -12,8 +12,6 @@
 OrderStatus.create(name: "Complete")
 OrderStatus.create(name: "Reconciled")
 
-Affiliate.find_or_create_by(name: "Other")
```

Comment:
The first reference to `Affiliate.OTHER` should create it. We could through in an `Affiliate.OTHER` reference to force it at seed time.